packages/contracts/contracts/README.md

```diff
@@ -0,0 +1,103 @@
+<p align="center">
+    <h1 align="center">
+        Semaphore Extension Contracts
+    </h1>
+    <p align="center">Extension contracts to manage Voting and whistleBlowing</p>
```

Comment:
The description should be more general, as there will be other extensions.

Something like "Solidity contracts to enhance Semaphore with extra features and customization."